content/en/blog/news/Mastering_Golang_Microservices_A_Practical_Guide_Embrace_High_Performance_with_Kitex_and_Hertz/index.md

```diff
@@ -0,0 +1,315 @@
+
+### Kitex & Hertz
+#### Prerequisites
+Before diving into CloudWeGo development with Kitex & Hertz, make sure you have set up the Golang development environment. Please follow the Install Go guide if you haven't already. 
+
+We highly recommend using the latest version of Golang, ensuring compatibility with three most recent minor release versions (currently >= v1.16). 
+
+Additionally, make sure that `GO111MODULE` is set to `ON`. 
+
+### Install the CLI tool
+
+Let's start by installing the CLI tools we will be working with.
+Ensure the `GOPATH` environment variable is properly defined (e.g., `export GOPATH=~/go`), then add `$GOPATH/bin` to the `PATH` environment variable (e.g., `export PATH=$GOPATH/bin:$PATH`). Make sure that `GOPATH` is accessible.
+
+Next, install Kitex (`go install github.com/cloudwego/kitex/tool/cmd/kitex@latest`), Thriftgo (for Thrift protocol - `go install github.com/cloudwego/thriftgo@latest`), and Hertz (`go install github.com/cloudwego/hertz/cmd/hz@latest`).
+
+Now, if you run `kitex --version`, `thriftgo --version`, and `hz --version`, you should see output indicating the versions of each CLI tool:
+
+```
+$ kitex --version
+vx.x.x
+$ thriftgo --version
+thriftgo x.x.x
+$ hz --version
+vx.x.x
+```
+**Note:** If you encounter any issues during the installation, it's likely due to gaps in the setup of the Golang development environment. Usually, you can quickly find a solution by searching for the error message online.
+
+## III. Creating A Sample Project
+### Kitex
+#### Get the example
+
+1. You can simply click [here](https://github.com/cloudwego/kitex-examples/archive/refs/heads/main.zip) to download the example.
+2. Or you can clone the sample repository `git clone https://github.com/cloudwego/kitex-examples.git`.
+
+#### Run the example
+##### Run with go
+1. Change to the `hello` directory. Hello is a simple example of Kitex using the Thrift protocol.
+`cd kitex-examples/hello`
+2. Run server
+`go run .`
+3. Run client
+open another terminal and `go run ./client.`
+
+##### Run with Docker
+1. Go to the examples directory
+`cd kitex-examples`
+2. Build the example project
+`docker build -t kitex-examples`.
+3. Run the server
+`docker run --network host kitex-examples ./hello-server`
+4. Run the client
+Open another terminal and run `docker run --network host kitex-examples ./hello-client`
+
+Congratulations! You now have successfully used Kitex to complete an RPC.
+
+### Hertz
+##### Quick Start
+To create a sample project with Hertz, start by creating the `hertz_demo` folder in the current directory and navigate to that directory. Then, create the `main.go` file and add the following code:
+package main
+
+```
```

Comment:
\```go

Reply:
Noted. Can you please double check the rest of the code blocks? 
The code blocks were copied 1 to 1 from your Feishu doc, so I'm not sure whether the language should be explicitly added on each block @GuangmingLuo